Dart/src/com/jetbrains/lang/dart/ide/formatter/DartBlock.java

```diff
@@ -184,4 +221,16 @@ public BlockWithParent getParent() {
   public void setParent(BlockWithParent newParent) {
     myParent = newParent;
   }
-}
\ No newline at end of file
+
+  private static class DartCascadeBlock extends DartBlock {
```

Comment:
For easier debugging we usually override toString(), in this case in DartBlock class. Adding empty inheritor for this purpose is too heavy solution :)